app/assets/stylesheets/friendships.css.scss

```diff
@@ -0,0 +1,16 @@
+.friendship {
+  border-bottom: 1px dotted #d3d3d3;
```

Comment:
Color literals like `#d3d3d3` should only be used in variable declarations; they should be referred to via variable everywhere else.